dotcom-rendering/src/static/icons/Star.tsx

```diff
@@ -1,11 +1,13 @@
-import { neutral } from '@guardian/source-foundations';
+import { palette } from '../../palette';
 
 export const Star = ({
 	starId,
 	isEmpty,
+	fill = palette('--star-rating-fill'),
```

Comment:
Why is this a property of this component? I think these values should either be set via custom properties or actual values, but not both.

If you wanted to change the fill colour of a star, you would manually set the `--star-rating-fill` property on an element or one of its ancestors.